### ISSUE TITLE
[Feature] Elementwise operator complex_tensor.normalize()

### ISSUE THREAD

**Issue by zasdfgbnm** [pytorch/pytorch#38074]
## 🚀 Feature

This elementwise operator converts each complex number `R * exp(i * theta)` to `exp(i * theta)`.

I am just feeling that having this operator will be convenient, and it is easy to implement such an operator using TensorIterator, and it could be implemented much faster than `t / t.abs()` on CUDA (native implementation of normalize has 1 memory read, 1 memory write, while `t/t.abs()` has 3 memory reads and 2 memory writes).

cc @ezyang @anjali411 @dylanbespalko

**Comment by vadimkantorov** [comment]
A naming consideration: if ever F.normalize gets moved to core ops (and even if not), there may be confusion

**Comment by anjali411** [comment]
Hey @zasdfgbnm great point! We [discussed](https://github.com/pytorch/pytorch/issues/36323#issuecomment-613569847) to add this feature as `torch.sgn`

**Comment by zasdfgbnm** [comment]
Closing this because there is already a discussion about `torch.sgn`.
